projects/vtk.cmake

```diff
@@ -18,6 +18,7 @@ superbuild_add_project(vtk
   DEPENDS_OPTIONAL ospray ${vtk_platform_dependencies}
   CMAKE_ARGS
     -DCMAKE_MACOSX_RPATH=OFF
+    -DKWSYS_NO_EXECINFO=ON
```

Comment:
Is it fixing the issue in netcdf?

Reply:
there is no issue in netcdf

Reply:
see this discussion: https://github.com/f3d-app/f3d-superbuild/pull/83/files#r1257313491